docs/products/clickhouse/howto/query-stats.rst

```diff
@@ -0,0 +1,22 @@
+Query statistics
+=================
+Usually, Query statistics in ClickHouse can be obtained using the ``system.query_log`` table, which stores statistics of each executed query, including memory usage and duration. Aiven for ClickHouse® currently do not give access to the table ``system.query_log`` table, in order to directly query the table for the required statistics.
```

Comment:
Consider rephrasing into:

Usually, query statistics in ClickHouse can be obtained using the ``system.query_log`` table, which stores statistics of each executed query, including memory usage and duration. In Aiven for ClickHouse®, the ``system.query_log`` table is currently not accessible for the purpose of obtaining query statistics.